packages/integrations/tailwind/README.md

```diff
@@ -146,14 +164,18 @@ module.exports = {
 }
 ```
 
-## Examples
+### Class-based directives like `dark:text-pink` with `@apply` does not work
 
-- The [Astro Tailwind Starter](https://github.com/withastro/astro/tree/latest/examples/with-tailwindcss?on=github) gets you up and running with a base for your project that uses Tailwind for styling
-- Astro's homepage uses Tailwind. Check out its [Tailwind configuration file](https://github.com/withastro/astro.build/blob/main/tailwind.config.cjs) or an [example component](https://github.com/withastro/astro.build/blob/main/src/components/IntegrationCard.astro)
-- The [Astro Ink](https://github.com/one-aalam/astro-ink), [Sarissa Blog](https://github.com/iozcelik/SarissaBlogAstroStarter), and [Creek](https://github.com/robertguss/Astro-Theme-Creek) themes use Tailwind for styling
-- [Browse Astro Tailwind projects on GitHub](https://github.com/search?q=%22%40astrojs%2Ftailwind%22+filename%3Apackage.json&type=Code) for more examples!
+Astro `<style>` tags are scoped by default. As `dark:text-pink` compiles into `.dark .text-pink`, it does not match any markup in the Astro file and is hence removed, causing the class to not be applied. This may also happen for integrations like Vue and Svelte that supports scoped styles.
```

Comment:
Reworked this slightly to move the example class down from the subheading.

```suggestion
Class names using modifiers like `dark:text-pink` or `focus:ring` are not compatible with the `@apply` directive in Astro `<style>` tags. For example, `dark:text-pink` compiles to `.dark .text-pink`, which is removed from build output because it does not match any markup in the `.astro` file. The same issue may also happen in framework components that support scoped styles like Vue and Svelte.
```

Reply:
Given the above, maybe this is more accurate? Are there any other good examples where this is the case? I guess the sample applies with the group classes like `group-hover:*`?

```suggestion
Certain Tailwind classes with modifiers rely on combining classes across multiple elements. For example, `dark:text-pink` compiles to `.dark .text-pink`. When this is used with the `@apply` directive in Astro `<style>` tags, the compiled styles are removed from the build output because they do not match any markup in the `.astro` file. The same issue may also happen in framework components that support scoped styles like Vue and Svelte.
```

Reply:
Yep `group-hover:*` should also have the same issue. I think that's a better example too as `dark:` could be configured to use media queries. I'll swap that out after committing your suggestion